docs/User-Guide_Autoconfig.md

```diff
@@ -4,90 +4,87 @@
 
 It is possible to configure your device automatically at first boot. Settings like: root password, IP address, connecting to wireless. 
 
-After flashing an image to boot media, mount it and add a file containing your config to /root/.not_logged_in_yet
+After flashing an image to boot media, mount it and add a file containing your config to `/root/.not_logged_in_yet`
 
-???+ info
-
-    This file will be read at your first login and configure system automatically.
-
-Config example for reference:
-
-```bash
-# Set PRESET_NET_CHANGE_DEFAULTS to 1 to apply any network related 
-# settings below
+???+ tip
+    You may also mount the image and edit it prior to flashing, if this is preferable.
+
+## Loading a remote config
+
+It is also possible to load this config file from a remote server, as above, however the **only** directive you should include is `PRESET_CONFIGURATION`  
+`PRESET_CONFIGURATION="http://path/to/config/file"`
+
+## Configuration directives
+
+
+- The directives in this file are specified using `key="value"` format.  
+- To ask for a value interactively, leave it unset or comment out the directive.  
+- For fully-unattended setup, specify all values  
+
+| Configuration directive | `[default] \| option` | Description: |
+| :---------------------- | :--------------: | -----------: |
+| `PRESET_CONFIGURATION` |  `http://path/to/config/file` |  See [Loading a remote config](#loading-a-remote-config) ||
+| `PRESET_NET_CHANGE_DEFAULTS` | `[0] 1` | Change default network settings, if unset, **no network changes will be applied** |
+| `PRESET_NET_ETHERNET_ENABLED` | `0 \| 1` | Enable Ethernet, ignored if WiFi enabled |
+| `PRESET_NET_WIFI_ENABLED` | `0 \| 1` | Enable WiFi, **takes priority over Ethernet** |
+| `PRESET_NET_WIFI_SSID` | `MySSID` | WiFi SSID |
+| `PRESET_NET_WIFI_KEY` | `MyWPA-PSK` | WiFi Pre-Shared Key (Password), **stored in plaintext** |
+| `PRESET_NET_WIFI_COUNTRYCODE` | `CC` | Country code, **required** for WiFi; e.g. `GB`, `US`, `DE`; see [Wikipedia/ISO_3166](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) |
+| `PRESET_NET_USE_STATIC` | `[0] 1` | Use the static IP provided, DHCP is the default; leaving **any** value unset will result in a broken config |
+| `PRESET_NET_STATIC_IP` | `xxx.xxx.xxx.xxx` | Static IPv4 address, dotted decimal notation |
+| `PRESET_NET_STATIC_MASK` | `xxx.xxx.xxx.xxx` | Subnet mask, typically `255.255.255.0` |
+| `PRESET_NET_STATIC_GATEWAY` | `xxx.xxx.xxx.xxx` | Default gateway
+| `PRESET_NET_STATIC_DNS` | `x.x.x.x x.x.x.x` | DNS Servers to use, separated by a space. If unsure, CloudFlare is `1.1.1.1 1.0.0.1`, Google is `8.8.8.8 8.8.4.4` |
+| `PRESET_USER_SHELL` | `shell` | Currently only `bash` (default) or `zsh` (`armbian-zsh`) supported |
+| `PRESET_CONNECT_WIRELESS` | `Y \| n` | Set to `Y` for interactive mode, `n` for automatic |
```

Comment:
Still not 100% this is the correct way round, but can't validate

Reply:
Perhaps saying "n" for reading preset values from this config?